projects/sticky-directive/src/lib/sticky.directive.ts

```diff
@@ -128,6 +134,10 @@ export class StickyDirective implements OnInit, AfterViewInit {
    * Start listening to the scroll event on the container element
    */
   private setObserver() {
+    if (isPlatformServer(this.platformId)) {
```

Comment:
Can you explain a little about this.

We think that works as follow:
- If it's on server side, does nothing
- If i'ts on the browser starts listening

> We know almost nothing about SSR

Reply:
That is correct, that is how it works.